routes/common/postgres_helper.rb

```diff
@@ -259,10 +259,12 @@ def failover
   end
 
   def send_notification_mail_to_partners(resource, user_email)
-    if resource.flavor == PostgresResource::Flavor::PARADEDB
-      Util.send_email(Config.postgres_paradedb_notification_email, "New ParadeDB Postgres database has been created.",
-        greeting: "Hello ParadeDB team,",
-        body: ["New ParadeDB Postgres database has been created.",
+    if [PostgresResource::Flavor::PARADEDB, PostgresResource::Flavor::LANTERN].include?(resource.flavor)
+      email = Config.send(:"postgres_#{resource.flavor}_notification_email")
+      flavor_name = resource.flavor.capitalize
+      Util.send_email(email, "New #{flavor_name} Postgres database has been created.",
+        greeting: "Hello #{flavor_name} team,",
+        body: ["New #{flavor_name} Postgres database has been created.",
           "ID: #{resource.ubid}",
```

Comment:
We might add the boot image version too

Reply:
This is is bit tricky. We don't know which version will be used at this point. It is decided by allocator and host availability.

Reply:
Fair point